src/wcs2_0/src/test/java/org/geoserver/wcs2_0/WCSTestSupport.java

```diff
@@ -252,6 +254,38 @@ protected void onSetUp(SystemTestData testData) throws Exception {
         namespaces.put("wcsgs", "http://www.geoserver.org/wcsgs/2.0");
         XMLUnit.setXpathNamespaceContext(new SimpleNamespaceContext(namespaces));
         xpath = XMLUnit.newXpathEngine();
+
+        // alter the native BBOX of one raster for at least a pixel to make sure the declared
+        // bounds are used, but also fitted to the grid to avoid resampling.
+        // Compared to the native bbox we are expanding by almost one pixel in all directions:
+        //          1 pixel
+        //             ↓
```

Comment:
```suggestion
        //             ↑
```

Reply:
Arrows is correct, but comment above is inconsistent, fixing.